carta/session.py

```diff
@@ -418,6 +418,57 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(String())
+    def set_viewer_mode(self, panel="single"):
+        """
+        Switch between single-panel mode and multiple-panel mode.
+
+        Parameters
+        ----------
+        panel : {0}
+            To adopt single-panel mode (``single``) or multiple-panel mode (``multiple``). Default is ``single``.
+        """
+        if panel is "single":
+            if_multiple = False
+        elif panel is "multiple":
+            if_multiple = True
+        else:
+            logger.warning('Cannot proceed with the input panel argument. Please set "single" or "multiple" for the argument.')
+            return
+        self.call_action("widgetsStore.setImageMultiPanelEnabled", if_multiple)
+
+    def previous_page(self):
+        """Go to previous page in viewer."""
+        self.call_action("widgetsStore.onPreviousPageClick")
+
+    def next_page(self):
+        """Go to next page in viewer."""
+        self.call_action("widgetsStore.onNextPageClick")
+
+    @validate(OneOf(*range(1, 101)), OneOf(*range(1, 101)), String())
```

Comment:
currently the frontend does not apply an upper limit (it should actually). I suggest we apply 10 here as an upper limit for now. Or to be consistent with the current behavior of the frontend, there is no upper limit in the validator. 

Can OneOf be used to validate "dynamic" and "fixed"? Then we don't need `if` checks.

Reply:
Yes, `OneOf` can validate "dynamic" and "fixed". In e62f8c4 it is used and I also used it to the `set_viewer_mode` function. Thanks a lot.